bin/protoc

```diff
@@ -9,11 +9,14 @@ def exec_protoc_for_os(os)
   case Platform::OS
     when :win32
       exec("#{File.dirname(__FILE__)}/protoc-#{Protoc::PROTOBUF_VERSION}-#{os}-#{PLATFORM_TO_PROTOC_ARCH[Platform::ARCH]}.exe", *ARGV)
-     when :aix
-      exec("LIBPATH=/opt/freeware/lib #{File.dirname(__FILE__)}/protoc-#{Protoc::PROTOBUF_VERSION}-#{os}-#{PLATFORM_TO_PROTOC_ARCH[Platform::ARCH]}", *ARGV)
     else
-      exec("#{File.dirname(__FILE__)}/protoc-#{Protoc::PROTOBUF_VERSION}-#{os}-#{PLATFORM_TO_PROTOC_ARCH[Platform::ARCH]}", *ARGV)
-  end
+      if os.eql? 'aix'
```

Comment:
Why wouldn't you just put this within the aix case statement? Its really weird to put another conditional in the fall through of the case statement.

Reply:
Why not use `Platform::OS` here

Reply:
There are a few things in the original code that I also think are oddly written, and could use refactoring.  I'm going for minimal changes here, and will probably refactor in the future.  🙂